### PR TITLE
integrated login and register

### PR DESCRIPTION
**Pull request by JustineDaveMagnaye**

### CODE REVIEW

File: user-management-application/src/main/java/com/user/management/data/employee/dao/impl/employeeDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import com.user.management.app.model.employee.Employee;
+import com.user.management.data.connection.ConnectionHelper;
+import com.user.management.data.employee.dao.employeeDao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class employeeDaoImpl implements employeeDao {
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/employee/dao/impl/employeeDaoImpl.java:13]: fix class naming convention

File: user-management-application/src/main/java/com/user/management/data/employee/dao/employeeDao.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.user.management.data.employee.dao;
+
+import com.user.management.app.model.employee.Employee;
+
+import java.sql.SQLException;
+
+public interface employeeDao {
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/employee/dao/employeeDao.java:7]: fix class naming convention

File: user-management-application/src/main/java/com/user/management/data/student/dao/impl/studentDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import com.user.management.app.model.student.Student;
+import com.user.management.data.connection.ConnectionHelper;
+import com.user.management.data.student.dao.studentDao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class studentDaoImpl implements studentDao {
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/student/dao/impl/studentDaoImpl.java:13]: fix class naming convention

File: user-management-application/src/main/java/com/user/management/data/student/dao/studentDao.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package com.user.management.data.student.dao;
+
+
+import com.user.management.app.model.student.Student;
+
+import java.sql.SQLException;
+
+public interface studentDao {
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/student/dao/studentDao.java:8]: fix class naming convention

File: user-management-application/src/main/java/com/user/management/app/facade/employee/Impl/EmployeeFacadeImpl.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        this.employeeDao = new employeeDaoImpl();
+    }
+
+    @Override
+    public Employee saveEmployee(Employee employee) throws SQLException {
+        return employeeDao.saveEmployee(employee);
+    }
+    @Override
+    public Employee checkEmployeeId(String employeeId) throws SQLException {
+        return employeeDao.checkEmployeeId(employeeId);
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/app/facade/employee/Impl/EmployeeFacadeImpl.java:27]: why is the name of the method like this? checkEmployeeId method implies to check if the employee id exists.
refactor accordingly. do the same with other methods.

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
check the comments. follow proper naming conventions and use appropriate method names

File: user-management-application/src/main/java/com/user/management/data/employee/dao/impl/employeeDaoImpl.java (hunk 1 above)
Comment (JustineDaveMagnaye) [reply user-management-application/src/main/java/com/user/management/data/employee/dao/impl/employeeDaoImpl.java]: I've fixed it sir

File: user-management-application/src/main/java/com/user/management/data/employee/dao/employeeDao.java (hunk 2 above)
Comment (JustineDaveMagnaye) [reply user-management-application/src/main/java/com/user/management/data/employee/dao/employeeDao.java]: I've fixed it sir

File: user-management-application/src/main/java/com/user/management/data/student/dao/impl/studentDaoImpl.java (hunk 3 above)
Comment (JustineDaveMagnaye) [reply user-management-application/src/main/java/com/user/management/data/student/dao/impl/studentDaoImpl.java]: I've fixed it sir

File: user-management-application/src/main/java/com/user/management/data/student/dao/studentDao.java (hunk 4 above)
Comment (JustineDaveMagnaye) [reply user-management-application/src/main/java/com/user/management/data/student/dao/studentDao.java]: I've fixed it sir

File: user-management-application/src/main/java/com/user/management/app/facade/employee/Impl/EmployeeFacadeImpl.java (hunk 5 above)
Comment (JustineDaveMagnaye) [reply user-management-application/src/main/java/com/user/management/app/facade/employee/Impl/EmployeeFacadeImpl.java]: I've refactored it accordingly sir

**Review by RoCS2024 — APPROVED** [review]

**Review by esdi-00 — APPROVED** [review]

**Pull request merged**